src/web/pages/base.ts

```diff
@@ -115,29 +115,35 @@ class Page {
     return waitForInvisibilityOf(this.getElement(elementName));
   }
 
-  public getElement(elementName: string) {
+  public getElement(elementName: any) {
```

Comment:
You union as a type, probably string | protractor.ElementArrayFinder

Reply:
Done